pkg/framework/aries/default.go

```diff
@@ -90,21 +85,26 @@ func defFrameworkOpts(frameworkOpts *Aries) error {
 		frameworkOpts.inboundTransport = inbound
 	}
 
+	didDBStore, err := frameworkOpts.storeProvider.OpenStore(peer.StoreNamespace)
```

Comment:
It seems a bit out of place to open a store for a specific method‘s namespace here.

Reply:
This will change when we add verifiable data registry (vdr)